src/content/docs/en/guides/testing.mdx

```diff
@@ -16,6 +16,115 @@ A Vite-native unit test framework with ESM, TypeScript and JSX support powered b
 
 See the [Astro + Vitest starter template](https://github.com/withastro/astro/tree/latest/examples/with-vitest) on GitHub.
 
+## Cypress
+
+Cypress is a next generation front end testing tool built for the modern web. Cypress enables you to write all types of tests:
+
+* End-to-end tests
+* Component tests
+* Integration tests
+* Unit tests
+
+Cypress can test anything that runs in a browser.
+
+### Installation
+
+You can install Cypress within your Astro project using the package manager of your choice.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install -D cypress
+  ```
+
+  This will install Cypress locally as a dev dependency for your project.
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add cypress --save-dev
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add cypress --dev
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuration
+
+In the root of your project, create a `cypress.config.ts` file with the following content:
```

Comment:
I get `Unknown file extension ".ts"` with this config setup. It's worth changing to `.js` to avoid this potential error in a getting started guide

```suggestion
In the root of your project, create a `cypress.config.js` file with the following content:
```

Reply:
> I get `Unknown file extension ".ts"` with this config setup. It's worth changing to `.js` to avoid this potential error in a getting started guide

It's strange, because I've tested my setup multiple times without any issues.

<img width="356" alt="cypress" src="https://user-images.githubusercontent.com/16765690/220197506-4898ef22-7c53-434a-95ed-8cde6b89f7c0.png">

So, in your case, changing the extension solved the error? I believe there should be an error related to using `import` in `.js` files and [according to the docs](https://docs.cypress.io/guides/references/configuration#Intelligent-Code-Completion), the config should be changed for js:

```js
const { defineConfig } = require('cypress')

module.exports = defineConfig({ ... })
```

Reply:
Yep, I get this error: 
<img width="641" alt="image" src="https://user-images.githubusercontent.com/4033662/220239060-2b58444a-468c-4fc6-bd84-fc2913323632.png">
And all I had to do was change to `.js`, no `require` necessary